### PR TITLE
Latest updates (minor)

### PR DESCRIPTION
**Pull request by ekarafezov**
Following updates are included:
1. Add MEC to supported Office prod channels
2. Add note to check oem guides for turning on virtualization
3. Add location for Office policy (user)
4. Clarify full telemetry is not required
5. Updated known issues

@Dansimp

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 1]
```diff
@@ -114,7 +118,9 @@ Upon being opened, the file should display a few visual indicators that the file
 
 ## Configure Application Guard for Office
 
-Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the Office cloud policy service.
+Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office Cloud Policy Service] (/DeployOffice/overview-office-cloud-policy-service).
```
Comment (schmurky) [inline microsoft-365/security/office-365-security/install-app-guard.md]: double-check in preview if the space between ] and ( affects render of hyperlink

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 2]
```diff
@@ -226,4 +232,5 @@ When this heuristic is met, Office will pre-create an Application Guard containe
 
 * Selecting web links (`http` or `https`) doesn't open the browser.
 * Pasting rich text format (RTF) content or images in Office documents opened with Application Guard isn't supported at this time.
-* Updates to .NET cause files to fail to open in Application Guard. As a workaround, users can restart their device when they come across this failure. Learn more about the issue at [Receiving an error message when attempting to open Windows Defender Application Guard or Windows Sandbox](https://support.microsoft.com/help/4575917/receiving-an-error-message-when-attempting-to-open-windows-defender-ap).
\ No newline at end of file
+* Default setting for unsuported file types protection policy is to block opening untrusted unsupported file types of IRM (Information Rights Management), CSV or HTML
```
Comment (schmurky) [inline microsoft-365/security/office-365-security/install-app-guard.md]: edit to:

Default setting for unsupported file types protection policy is to block opening untrusted and/or unsupported file types including IRM (Information Rights Management), CSV or HTML.

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 3]
```diff
@@ -226,4 +232,5 @@ When this heuristic is met, Office will pre-create an Application Guard containe
...
+* Updates to .NET cause files to fail to open in Application Guard. As a workaround, users can restart their device when they come across this failure. Learn more about the issue at [Receiving an error message when attempting to open Windows Defender Application Guard or Windows Sandbox](https://support.microsoft.com/help/4575917/receiving-an-error-message-when-attempting-to-open-windows-defender-ap).
```
Comment (schmurky) [inline microsoft-365/security/office-365-security/install-app-guard.md]: edit to:

Updates to .NET can cause files to fail to open in Application Guard.

also, can't verify if there is a space between ] and ( for the hyperlink. if there is, kindly remove.

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 4]
```diff
@@ -114,7 +118,9 @@ Upon being opened, the file should display a few visual indicators that the file
 
 ## Configure Application Guard for Office
 
-Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the Office cloud policy service.
+Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office Cloud Policy Service] (/DeployOffice/overview-office-cloud-policy-service).
+See configuration set by your administrator by reviewing group policy settings in  **User Configuration\\Administrative Templates\\Microsoft Office 2016\\Security Settings\\ Trust Center\\Application Guard**.
```
Comment (schmurky) [inline microsoft-365/security/office-365-security/install-app-guard.md]: remove space before Trust

**Review by schmurky — CHANGES_REQUESTED** [review]

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 5]
```diff
@@ -37,10 +37,11 @@ Microsoft Defender Application Guard for Office (Application Guard for Office) h
...
-* **Office**: Office Current Channel Build version 2011 16.0.13530.10000 or later. Both 32-bit and 64-bit versions of Office are supported.
+* **Office**: Office Current Channel and Monthly Enterprise Channel, Build version 2011 16.0.13530.10000 or later. Both 32-bit and 64-bit versions of Office are supported.
 * **Update package**: Windows 10 cumulative monthly security update [KB4571756](https://support.microsoft.com/help/4571756/windows-10-update-KB4571756)
 
-For detailed system requirements, refer to [System requirements for Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/reqs-md-app-guard). To learn more about Office update channels, see [Overview of update channels for Microsoft 365](/deployoffice/overview-update-channels).
+For detailed system requirements, refer to [System requirements for Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/reqs-md-app-guard). Please refer to your computer manufacturer guides on how to enable virtualization technology.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
For detailed system requirements, refer to [System requirements for Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/reqs-md-app-guard). Also, please refer to your computer manufacturer's guides on how to enable virtualization technology.
```

File: microsoft-365/security/office-365-security/install-app-guard.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office Cloud Policy Service](/DeployOffice/overview-office-cloud-policy-service).
```

File: microsoft-365/security/office-365-security/install-app-guard.md (hunk 4 above)
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
See configuration set by your administrator by reviewing group policy settings in  **User Configuration\\Administrative Templates\\Microsoft Office 2016\\Security Settings\\Trust Center\\Application Guard**.
```

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 6]
```diff
@@ -182,13 +188,15 @@ You may also submit feedback from within Office if the issue happens when Office
 
 Application Guard for Office is integrated with Microsoft Defender for Endpoint to provide monitoring and alerting on malicious activity that happens in the isolated environment.
 
+[Safe Documents in Microsoft E365 E5] (/microsoft-365/security/office-365-security/safe-docs) is a feature that uses Microsoft Defender for Endpoint to scan documents opened in Application Guard for Office. For an additional layer of protection, users can't leave Applicatio Guard for Office until results of the scan have been determined.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
[Safe Documents in Microsoft E365 E5](/microsoft-365/security/office-365-security/safe-docs) is a feature that uses Microsoft Defender for Endpoint to scan documents opened in Application Guard for Office. For an additional layer of protection, users can't leave Application Guard for Office until the results of the scan have been determined.
```

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 7]
```diff
@@ -226,4 +234,5 @@ When this heuristic is met, Office will pre-create an Application Guard containe
 
 * Selecting web links (`http` or `https`) doesn't open the browser.
 * Pasting rich text format (RTF) content or images in Office documents opened with Application Guard isn't supported at this time.
-* Updates to .NET cause files to fail to open in Application Guard. As a workaround, users can restart their device when they come across this failure. Learn more about the issue at [Receiving an error message when attempting to open Windows Defender Application Guard or Windows Sandbox](https://support.microsoft.com/help/4575917/receiving-an-error-message-when-attempting-to-open-windows-defender-ap).
\ No newline at end of file
+* Default setting for unsuported file types protection policy is to block opening untrusted unsupported file types of IRM (Information Rights Management), CSV or HTML
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
* The default setting for unsupported file types protection policy is to block opening untrusted unsupported file types of Information Rights Management (IRM), CSV, or HTML.
```

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 8]
```diff
@@ -226,4 +234,5 @@ When this heuristic is met, Office will pre-create an Application Guard containe
...
+* Updates to .NET cause files to fail to open in Application Guard. As a workaround, users can restart their device when they come across this failure. Learn more about the issue at [Receiving an error message when attempting to open Windows Defender Application Guard or Windows Sandbox](https://support.microsoft.com/help/4575917/receiving-an-error-message-when-attempting-to-open-windows-defender-ap).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
* Updates to .NET might cause files to fail to open in Application Guard. As a workaround, users can restart their device when they come across this failure. Learn more about the issue at [Receiving an error message when attempting to open Windows Defender Application Guard or Windows Sandbox](https://support.microsoft.com/help/4575917/receiving-an-error-message-when-attempting-to-open-windows-defender-ap).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 9]
```diff
@@ -114,7 +118,9 @@ Upon being opened, the file should display a few visual indicators that the file
 
 ## Configure Application Guard for Office
 
-Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the Office cloud policy service.
+Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office Cloud Policy Service](/DeployOffice/overview-office-cloud-policy-service).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office cloud policy service](/DeployOffice/overview-office-cloud-policy-service).
```

**Comment by yogkumgit** [comment]
@ekarafezov please implement the feedback. Thanks

**Comment by JohanFreelancer9** [comment]
@ekarafezov - please commit the rest of my suggested changes. Thanks.

File: microsoft-365/security/office-365-security/install-app-guard.md (hunk 7 above)
Comment (ekarafezov) [reply microsoft-365/security/office-365-security/install-app-guard.md]: There isn't a condition for "and/or". I like the updates but without the and/or. 
"The default setting for unsupported file types protection policy is to block opening untrusted unsupported file types of Information Rights Management (IRM), CSV, or HTML"

Comment (JohanFreelancer9) [reply microsoft-365/security/office-365-security/install-app-guard.md]: Thank you for your feedback. I have edited my suggestion accordingly. Could you please commit this final suggestion? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@schmurky Please merge this P. Thanks

**Pull request merged**